lib/tether/tether_linux.go

```diff
@@ -228,14 +227,25 @@ func establishPty(session *SessionConfig) error {
 		// it frees up all resources - does that mean it frees the output buffers?
 		go func() {
 			_, gerr := io.Copy(session.Outwriter, session.Pty)
-			log.Debugf("PTY stdout copy: %s", gerr)
+			if e, ok := gerr.(*os.PathError); ok && e.Err == syscall.EIO {
+				// We can safely ignore this error, because
+				// it's just the PTY telling us that it closed all good.
+				log.Debugf("Ignoring EIO error returned by Outwriter io.Copy")
```

Comment:
If we're explicitly logging a different message because this is expected we may as well it actually distinct:
"PTY copy finished"

and make the other case a Warn at least given it's not the expected path.